Assets/MRTK/Tests/PlayModeTests/Experimental/ObjectManipulatorTests.cs

```diff
@@ -1323,6 +1323,71 @@ public IEnumerator CursorContextObjectManipulatorMove()
             UnityEngine.Object.Destroy(cursorContextManipulationHandler);
             UnityEngine.Object.Destroy(manipulationHandler);
         }
+
+
+        /// <summary>
+        /// TODO: This test should move into base cursor tests once object manipulator graduates
+        /// Tests that the gaze cursor stays centered lockCursorWhenFocusLocked is toggled off
+        /// </summary>
+        [UnityTest]
+        public IEnumerator ObjectManipulatorGazeCursorFocusLock()
+        {
+            var inputSystem = PlayModeTestUtilities.GetInputSystem();
+
+            var iss = PlayModeTestUtilities.GetInputSimulationService();
+            var oldHandSimMode = iss.HandSimulationMode;
+            iss.HandSimulationMode = HandSimulationMode.Gestures;
+
+            // Setting this to false is equivalent to toggling lockCursorWhenFocusLocked off
+            inputSystem.GazeProvider.GazePointer.IsTargetPositionLockedOnFocusLock = false;
```

Comment:
Rather than doing this, could we instead explicitly set lockCursorWhenFocusLocked = false in code?

Similarly, can we test lockCursorWhenFocusLocked  = true to make sure the old behavior of locking works as well?

Reply:
Will add a test for the old behavior. Otherwise following keveleigh's above comment